src/source/Ice/TurnConnection.c

```diff
@@ -1409,6 +1409,7 @@ PCHAR turnConnectionGetStateStr(TURN_CONNECTION_STATE state)
         case TURN_STATE_FAILED:
             return TURN_STATE_FAILED_STR;
     }
+    return NULL;
```

Comment:
Recommend defining a new string TURN_STATE_UNKNOWN_STR in https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/e38951f4a9329e9e2d1c0b51cc0b41ef3b950e57/src/source/Ice/TurnConnection.h#L47 with something like "TURN_STATE_UNKNOWN"

Reply:
Sure, love to do it and I prefer that way. For every commits I make, I will draft one basic coding style first, and want to iterate these commits with you in order to fit the coding style and code convention of this sdk. It will be a little annoying. so thanks for your patience to review my commits. :)